### PR TITLE
WD-14152 - Add link to free trial contact form in /pro/subscribe

### PR DESCRIPTION
**Pull request by fasih-mehmood**
## Done

- Added link to [Free Trial Contact Form](https://ubuntu.com/engage/Ubuntu-Pro-1-month-trial) on the Free Trial Available button.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/pro/subscribe
    - Be sure to test on mobile, tablet and desktop screen sizes
- Scroll down to the "**Buy Now**" button and click the Free trial available button. You should be redirected to the free trial contact form.
- Try logging out if you can't see the "Free trial available" button.

## Issue / Card

Fixes [WD-14152](https://warthogs.atlassian.net/browse/WD-14152)

## Screenshots

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-14152]: https://warthogs.atlassian.net/browse/WD-14152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx [hunk 1]
```diff
@@ -203,9 +205,11 @@ const ProductSummary = () => {
...
           {product?.canBeTrialled && productUser !== ProductUsers.myself ? (
             <Col size={12}>
-              <StatusLabel appearance="positive">
-                Free trial available
-              </StatusLabel>
+              <a href="/engage/Ubuntu-Pro-1-month-trial">
+                <StatusLabel appearance="positive">
+                  Free trial available
+                </StatusLabel>
+              </a>
```
Comment (minkyngkm) [inline static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx:212]: It's working fine, but I noticed it doesn't look clickable, so users might not realize it is.  Should we move the `a` tag inside? What do you think?
```suggestion
          <StatusLabel appearance="positive">
              <a href="/engage/Ubuntu-Pro-1-month-trial">
                  Free trial available
              </a>
            </StatusLabel>
```

File: static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx [hunk 2]
```diff
@@ -120,9 +120,11 @@ const ProductSummary = () => {
...
           >
             {product?.canBeTrialled && productUser !== ProductUsers.myself ? (
-              <StatusLabel appearance="positive">
-                Free trial available
-              </StatusLabel>
+              <a href="/engage/Ubuntu-Pro-1-month-trial">
+                <StatusLabel appearance="positive">
+                  Free trial available
+                </StatusLabel>
+              </a>
```
Comment (minkyngkm) [inline static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx:127]: This part as well

File: static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx (hunk 1 above)
Comment (fasih-mehmood) [reply static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx:212]: how does this look?

image

Comment (minkyngkm) [reply static/js/src/advantage/subscribe/react/components/ProductSummary/ProductSummary.tsx:212]: Yep, this looks better for me

**Review by minkyngkm — APPROVED** [review]
After addressing my comments, its good to go.

**Pull request merged**